Initialtask/src/Validator/ProductValidator.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Validator;
+
+use App\Entity\Product;
+
+class ProductValidator
+{
+    public static function validate(array $productData): bool
+    {
+        $cost = (float) preg_replace('/[^0-9.]/', '', $productData['Cost in GBP']);
```

Comment:
why price is float
We discussed earlier, that it should be integer
Could you please fix it?